web/src/index.scss

```
@@ -94,28 +177,855 @@ body {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 8px;
+  font-size: 12px;
+  color: var(--ink-4);
+}
+
+/* Redeem note under the swap summary — ported from market.html */
+.return-note {
+  margin: 8px 2px 0;
+  font-family: "Inter", "IBM Plex Mono", ui-monospace, "SF Mono", Menlo, sans-serif;
```

Stylelint (17.13.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)